xyz

```diff
@@ -220,7 +220,7 @@ run npm prune
 run npm test
 
 for script in "${scripts[@]}" ; do
-  [[ $script == /* ]] || script="$(pwd)/$script"
+  [[ $script =~ ^/ ]] || script="$(pwd)/$script"
```

Comment:
Although this change is not necessary, using `=~` is consistent with the way in which matching is performed elsewhere in the program (`[[ $increment =~ ^pre ]]`, for example).